bin/run-single-tv-datasetcreation-pipeline.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
I know.  I thing I got the v from in vivo. My brain is dumb.

On Aug 1, 2016 9:40 AM, "akmorrow13" <notifications@github.com> wrote:

> In bin/run-single-tv-datasetcreation-pipeline.sh
> <https://github.com/akmorrow13/endive/pull/1#discussion_r73010397>:
>
> > @@ -0,0 +1,8 @@
> > +#!/bin/bash
>
> how about TF? (transcription factor = TF)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/akmorrow13/endive/pull/1/files/b00dbd128239b200b96c55e928cc42abdd9b4805#r73010397>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABTEkP2i9wQfh5VfBHdxCmaLKdQW1rbxks5qbiFfgaJpZM4JZw3M>
> .
>